stingray/utils.py

```diff
@@ -298,9 +298,18 @@ def is_iterable(stuff):
 
 
 def order_list_of_arrays(data, order):
+    """Convert data to a list of arrays.
```

Comment:
It is *not* designed to convert data into lists of arrays, but to sort arrays (alone or inside dicts) according to the order specified in `order`

Reply:
Fixed it.